learn/tanssi/staking.md

```diff
@@ -0,0 +1,119 @@
+
+![High-level overview of the Auto-Compounded Rewards Pool when Staking](/images/learn/tanssi/staking/staking-4.webp)
+
+Native token rewards are automatically assigned as new stake into the Auto-Compounded Rewards Pool, hence the auto-compounding nature of this specific staking pool mechanism.
+
+Nevertheless, when auto-compounded staking rewards are assigned, they are not held in the user's reserved balance, as the protocol-owned account still has them. The increase in the delegator's stake is indirectly represented by the share price increase. However, in specific scenarios, a user might want to let the protocol know that they want that balance to be represented in their state as reserved balance, for example, for governance purposes. 
+
+Consequently, the protocol offers a specific transaction any user can submit to update the reserve balance of any delegate. This call moves the auto-compounded rewards for the specified user from the protocol-owned account to their reserve balance. This is also automatically executed by the protocol when a user removes liquidity from a Auto-Compounded Rewards Pool.
+
+### Leaving Pool {: #leaving-pool}
+
+When a user decides to exit their staking positions from a Manual or Auto-Compounded Reward Pool, they have the power to initiate an undelegation. This process, similar to when they initially entered the Joining Pool, is a two-step journey. The user signs an intent to remove a specific delegation and patiently waits for at least one entire session before the operation can be executed by anyone.
+
+Upon executing the leaving transaction intent, the protocol exchanges shares of the specified pool for native tokens at the current pool price. For the Manual Rewards Pool, any unclaimed rewards are assigned to the user. Simultaneously, the protocol purchases Leaving Pool shares in a one-to-one ratio for the native tokens the user just received. This ensures that the user joins the Leaving Pool, acquiring shares that correspond to the number of native tokens they desire to unstake.
+
+After an entire session passes, any user can execute the pending operation. Then, the protocol swaps Leaving Pool shares for native protocol tokens at a one-to-one ratio.
+
+The primary purpose of the Leaving Pool is to provide a buffer for users leaving the staking mechanics. Consequently, through this buffer, the protocol can implement a slashing mechanism that affects undelegating users, but their stake was active during the slashable event. Nevertheless, slashing has not been implemented in Tanssi.
```

Comment:
This paragraph is a bit confusing at the end, mainly this part: 

> the protocol can implement a slashing mechanism that affects undelegating users, but their stake was active during the slashable event.

 I would simplify to slashing has not been implemented in Tanssi but could be implemented in the future.

Reply:
What about:

```
The primary purpose of the Leaving Pool is to provide a buffer for users leaving the staking mechanics. This buffer allows to implement slashing mechanisms to deter bad behavior, but this has not been implemented yet.
```

The Leaving Pool's only purpose is for slashing haha